### DISCUSSION TITLE
Can please help me

### DISCUSSION THREAD

**Discussion by ghost** [community/community#4936] [Other Feature Feedback, Questions, & Ideas] [answered]
My system is not working properly

**Comment by ghost** [comment]
I think it's hacked or bugged

**Reply by fedroleo** [reply]
i computer li hanno distribuiti assicurandosi di poterli poi controllare quando sono in giro per il mondo, mettiti al loro stesso piano, distribuiresti una macchina tecnologica cosi senza poterla più sorvegliare, se finisce in mani sbagliate sono problemi, camera e microfoni sempre attivi? 
Vi sono poi le frequenze?
Qui su github ci  vogliono le basi minime di conoscenza, non riesco a capire come procedere, ci vorrebbe un maestro di supporto, ciao buon tempo

**Comment by ghost** [comment]
Remove all my api’s and reset all my accounts and balance to zero. Since, I was bugged and hacked.

Thank you I really appreciate all the hard work and effort.

**Reply by fedroleo** [reply]
ciao buona notte

**Comment by ghost** [comment, marked as answer]
Thanks for the help and efforts, please remove all api's and reset my accounts and balance to zero (0). Since, I was bugged or hacked.
